packages/prisma/index.ts

```diff
@@ -59,6 +59,10 @@ const prismaWithClientExtensions = prismaWithoutClientExtensions
 
 export const prisma = globalForPrisma.prismaWithClientExtensions || prismaWithClientExtensions;
 
+export const readonlyPrisma = customPrisma({
+  datasources: { db: { url: process.env.INSIGHTS_DATABASE_URL } },
```

Comment:
We need to check and make sure this env variable is present and has data, otherwise, it should fall back to the DATABASE_URL. This helps self-hosters upgrade and not run into problems and also not have to install a separate DB.

Reply:
Done!